Update create_graph.py

seigodev opened the pull request:
Add assign_top_node() and assign_level2node_recursion().
階層割当を行う関数を追加。

seigodev committed:
Update create_graph.py

Add assign_top_node() and assign_level2node_recursion().
階層割当を行う関数を追加。

aabaa commented on demo/create_graph.py:
```diff
@@ -81,7 +81,54 @@ def create_node_list(input_node_dict):
+
+    """
+    for top_node in node_list:
+        if set() == top_node.target_nodes:
```
if not top_node.target_nodes:
のほうが読みやすいです

aabaa commented on demo/create_graph.py:
```diff
@@ -81,7 +81,54 @@ def create_node_list(input_node_dict):
+            assign_level2node_recursion(node_list, top_node, 0)
+
+
+def assign_level2node_recursion(node_list, target, target_level):
```
recursionは名詞なので副詞にします
assign_level2node_recursively

aabaa commented on demo/create_graph.py:
```diff
@@ -81,7 +81,54 @@ def create_node_list(input_node_dict):
+        elif assign_node.x > -1 and assign_node.y <= assign_node_level:
+            assign_node.y = assign_node_level
+        else:
+            pass
```
不要では

seigodev committed:
Fix requested changes.

・if set() == top_node.target_nodes: → if not top_node.target_nodes: (102行)
・assign_level2node_recursion() → assign_level2node_recursively() (105, 108, 125行)
・assign_level2node_recursively()のelseの除去(128, 129行)。
